weblab/entities/views.py

```diff
@@ -239,33 +242,63 @@ def get_context_data(self, **kwargs):
         kwargs['delete_file'] = self.request.GET.get('deletefile')
         return super().get_context_data(**kwargs)
 
-    def post(self, request, *args, **kwargs):
-        entity = self.get_object()
+    def add_and_delete_files(self, deletions, additions):
+        pass
```

Comment:
Was this supposed to be implemented? 😄

Reply:
Oops... I had a refactor in mind but decided against it for now... Removed.